server/game/cards/characters/04/robbstark.js

```diff
@@ -0,0 +1,58 @@
+const DrawCard = require('../../../drawcard.js');
+
+class RobbStark extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => (
+                this.game.currentChallenge &&
+                this.game.currentChallenge.challengeType === 'military'
+            ),
+            match: this,
+            effect: ability.effects.modifyStrength(this.numberOfLoyalChars())
+        });
+
+        this.action({
+            title: 'Initiate ability',
```

Comment:
I asked you about this in the chat a couple of days ago, where I believed you echoed my sentiments. :P Personally I would love it if any action was titled 'Initiate ability' like here, whether it is on Nymeria Sand, Iron Fleet Scout or Selyse Baratheon. I don't like the labels used currently, mostly from the 'Kneel this card to " variety. They look too much like 'Select this card', they are usually incomplete because of space constraints (like here, no room to mention non-King constraint) and they're sometimes just plain wrong (Iron Fleet Scout when you're first player for example). Having them all named the same is nice from a templating perspective, it stands out from 'Select this card' and if you don't know what the ability does, the card is shown large since you're mouse-over at that point anyway. Your call though. :)